lux/executor/PandasExecutor.py

```diff
@@ -538,11 +531,17 @@ def _is_datetime_number(series):
         return False
 
     def compute_stats(self, ldf: LuxDataFrame):
+        # use sample to compute statistics
+        if ldf._sampled is None:
+            ldf_sampled = PandasExecutor.execute_sampling(ldf)
```

Comment:
Will the config parameters that we are using for sampling for metadata and the visualization be the same?

Reply:
Yes, currently they are the same (`sampling_thresh`). Should we maybe use different parameters?